.github/workflows/prepare_for_release.yml

```diff
@@ -0,0 +1,31 @@
+name: Reusable action of Release preparation
+
+on: 
+  workflow_call:
+    secrets:
+      CI_USER_TOKEN:
+        required: true
+
+jobs:
+  prepare_for_release:
+    runs-on: macos-latest
+    steps:
+    - uses: maxim-lobanov/setup-xcode@v1
+    - uses: actions/checkout@v3
+      with:
+        xcode-version: '12.4'
+    - id: prepare_for_release
+      name: Prepare for release
+      env:
+        VERSION: 3.10.1
```

Comment:
I searched about it and have tried to make it a global env. For using `reusable_workflows`, we cannot pass env variables to them.  Because of this, version is declared twice in `swift.yml`.